### DISCUSSION TITLE
Alternative library search

### DISCUSSION THREAD

**Discussion by y0shir** [audacious-media-player/audacious#113] [Ideas]
Current library search only displays up to 20 items per search, which while decent enough for most cases, is bothersome when you have authors and singers with big amount of albums. It also requiters you to remember how exactly their name is spelled, which can be inconvenient.
Would alternative system, where without any query you are presented with list of authors, from which you can go to list of albums of selected author, from which you can go to specific song to that author be better?

**Comment by jlindgren90** [comment]
You can increase the limit of 20 to a larger value in the Search Tool plugin settings, for what it's worth.

**Reply by Hideman85** [reply]
Thanks, I was in the exact same issue, actually my usecase was a bit different, I was searching for an album tag and I was seeing 20 out of 125 where I wanted to add them all to the current playlist but it wasnt. Now with 200 I can add them all..

BTW could be worth like category, author, to have when we search a grouping on album, I would have been able to see and add Album Name 125 songs directly.
